src/profile/webid.ts

```diff
@@ -76,7 +76,9 @@ export function getAltProfileUrlAllFrom(
  * A WebID Profile may be any RDF resource on the Web, it doesn't have
  * to be a Solid resource. That is why, in order to expose a Solid-enabled part
  * of their profile, some WebID profiles link to a Profile Resource, which may
- * be a Solid resource.
+ * be a Solid resource. A WebID resource should be public, so `getProfileAll` will
```

Comment:
So ... since we state "to expose a Solid-enabled part ..." above, should  "a Profile Resource, which may be a Solid resource"  be just  "a Profile Resource, which is ..."?  Or  even  "an Extended Profile, which is a Solid Resource"?  

And, with "WebID resource should be public" -- is it a should be or is it a must?  If a must, I'd just say "A WebID profile is publicly readable"

Reply:
Oh ... also, in case your "should be public" stands -- such that, some people make their profiles not publicly readable (mistakenly or not),  just wondering, for the WebID profile part of getProfileAll, do we first do an unauthed fetch, and then, if that fails, we do an authe'd fetch?

Reply:
"Do we first do a...", yes, that's the implementation: https://github.com/inrupt/solid-client-js/pull/1709/files#diff-df051b184aa88ad98045d31b7def5698e9b5c75b17fd85b15bc8a8790cf82798R122

keep in mind this isn't strictly specified by a spec yet.